### DISCUSSION TITLE
Facebook Integration stuck on loading

### DISCUSSION THREAD

**Discussion by dayanidhi-dehury** [chatwoot/chatwoot#6049] [Self hosted]
Not able to Integrate Messenger into Chatwoot

When we are trying to integrate the messenger into Chatwoot the login page for Facebook is not opening it keeps on loading on the authentication page. 

**To Reproduce**

Steps to reproduce the behaviour:
1. Go to 'Settings of chatwoot'
2. Click on 'Inboxes'
3. Click on 'Add Inbox' in the top right corner.
4. From Choose a channel, Click on 'Messenger'.
5. Click on 'Continue with facebook'
7. See the error

**Expected behavior**

It should show the Facebook login window to enter the credentials.

**Screenshots**
![image](https://user-images.githubusercontent.com/120084890/206398656-0327d6a4-4ecb-474e-9709-084e2e57a2c0.png)

**Browser logs**

**Environment**

Describe whether you are using Chatwoot Cloud (app.chatwoot.com) or a self-hosted installation of Chatwoot. If you are using a self-hosted installation of Chatwoot, describe the type of deployment (Docker/Linux VM installation/Heroku/Kubernetes/Other).

- [ ] app.chatwoot.com (Chatwoot Cloud)
- [x] Self-hosted
- - [ ] Linux VM
- - [ ] Docker
- - [ ] Kubernetes
- - [ ] Heroku
- - [ ] Other (Please specify)

**Desktop (please complete the following information)** (If applicable)
 - OS: [Windows 10 Home Single Language - 21H2]
 - Browser [Firefox ]
 - Version [107.0.1 (64-bit)]

**Cloud Provider** (If applicable)
- [ ] AWS
- [ ] GCP
- [ ] Azure
- [ ] DigitalOcean
- [ ] Others

**Comment by pranavrajs** [comment]
@dayanidhi-dehury Moving this to discussions as this most likely is not a bug. Please check the documents here and make sure that you have gotten developer permissions from Facebook. 

https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/facebook-channel-setup

**Reply by dayanidhi-dehury** [reply]
Hey @pranavrajs 

Thanks for getting back, I will check with the team and let you know.
